client/clientmanager/src/main/java/io/mosip/registration/clientmanager/service/MasterDataServiceImpl.java

```diff
@@ -209,6 +214,89 @@ public void onFailure(Call<ResponseWrapper<ClientSettingDto>> call, Throwable t)
         });
     }
 
+    @Override
+    public void syncGlobalParamsData() throws Exception {
+
+        Log.i(TAG, "config data sync is started");
+
+        Call<ResponseWrapper<Map<String, Object>>> call = syncRestService.getGlobalConfigs(clientCryptoManagerService.getClientKeyIndex());
+
+        call.enqueue(new Callback<ResponseWrapper<Map<String, Object>>>() {
+            @Override
+            public void onResponse(Call<ResponseWrapper<Map<String, Object>>> call, Response<ResponseWrapper<Map<String, Object>>> response) {
+                if (response.isSuccessful()) {
+                    ServiceError error = SyncRestUtil.getServiceError(response.body());
+                    if (error == null) {
+                        saveGlobalParams(response.body().getResponse());
+                        Toast.makeText(context, "Global config sync Completed", Toast.LENGTH_LONG).show();
+                    } else
+                        Toast.makeText(context, "Global config sync failed " + error.getMessage(), Toast.LENGTH_LONG).show();
+                } else
+                    Toast.makeText(context, "Global config sync failed with status code : " + response.code(), Toast.LENGTH_LONG).show();
+            }
+
+            @Override
+            public void onFailure(Call<ResponseWrapper<Map<String, Object>>> call, Throwable t) {
+                Toast.makeText(context, "Global config sync failed", Toast.LENGTH_LONG).show();
```

Comment:
Add these constants in the strings.xml under values to support internationalization.